03-Purpose-and-Principles/Documentation-Principles.md

```diff
@@ -0,0 +1,46 @@
+# Documentation Principles
+
+These principles are not set in stone -- if you'd like clarification, qualification, addition, removal, or changes to these principles, open an issue to discuss.
+
+## Documentation Scope
+
+- There are many types of applications that can be built with PureScript, each of which should have documentation. The group of people who write each type of application is best suited for managing its documentation.
```

Comment:
Doesn't this say more about 'who' should be writing the docs for something rather than the scope of documentation / what to cover and not to cover?

Reply:
I intended this to imply the scope of the documentation. The audience of a piece of documentation is affects the scope, which is why I felt it important to speak about that here. You're right that this should be more explicit about the reason the people side of this is introduced. I'll change this to clarify that.

Reply:
Addressed by https://github.com/chexxor/purescript-documentation-discussion/pull/69/commits/ef38c4a06726eb5b95ebcfb2fb96044587bcf8b2